include/blade/pipelines/generic/file_reader.hh

```diff
@@ -124,6 +124,10 @@ class BLADE_API FileReader : public Pipeline {
         return bfr5->getBeamCoordinates();
     }
 
+    constexpr Modules::Guppi::Reader<OT>& getGuppi() {
+        return *guppi;
+    }
```

Comment:
Do we need this?

Reply:
Nope, I've removed it. I've been using it for debugging-prints